query/src/main/scala/filodb/query/LogicalPlan.scala

```diff
@@ -518,6 +685,14 @@ case class ApplyInstantFunctionRaw(vectors: RawSeries,
     vectors = vectors.replaceRawSeriesFilters(newFilters).asInstanceOf[RawSeries],
     functionArgs = functionArgs.map(_.replacePeriodicSeriesFilters(newFilters).asInstanceOf[FunctionArgsPlan]))
 
+  override def useHigherLevelAggregatedMetricIfApplicable(params: HierarchicalQueryExperience): RawSeriesLikePlan = {
+    // TODO: use raw series
+    this.copy(
```

Comment:
@amolnayak311 @alextheimer @sherali42  what do you think about this case ? Should we update raw filter in this case ?

Reply:
I allowed the update here to keep it consistent with other `RawSeriesLikePlan` updates

Reply:
This is now gated with a config and disabled by default to limit the changes.